src/js/button.js

```diff
@@ -28,25 +28,11 @@ class Button extends ClickableComponent {
    * @return {Element}
    *         The element that gets created.
    */
-  createEl(tag = 'button', props = {}, attributes = {}) {
+  createEl(props = {}, attributes = {}) {
```

Comment:
After some thought, I'm unsure whether we want to change the method signature altogether. It totally makes sense to me to ignore it and always create a button element but it feels weird for this method to have a different signature from all other `createEl`s.